docs/IE_DG/Extensibility_DG/AddingNGraphOps.md

```diff
@@ -51,6 +53,12 @@ nGraph operation contains two constructors: a default constructor, which allows
 
 @snippet op.cpp op:visit_attributes
 
+### `evaluate()`
+
+`ngraph::Node::evaluate` method allows to apply constant folding for operation.
```

Comment:
--> to apply constant folding to an operation

Reply:
Fixed